lib/scss/components/filter-menu/_filter-menu.scss

```diff
@@ -45,15 +47,15 @@
       float: right;
     }
 
-    @include grid-media-between('xs', 'lg') {
+    @include grid-media-max('lg') {
       &:before {
         @include small-paragraph;
         @include padding(null 15px null null);
 
         color: $su-color-bright-blue;
         content: "\f1de"; // Placeholder Icon until SU ones in place.
         font-family: 'Font Awesome 5 Free';
-        font-weight: 600; // Need this for the new version of FA. Will go away later.
+        font-weight: $su-font-semi-bold; // Need this for the new version of FA. Will go away later.
```

Comment:
Most of the font-weights we have use variables from Decanter.

Reply:
Ok, that's good to know.